pom.xml

```diff
@@ -16,6 +16,123 @@
     <version>1.1.3</version>
   </parent>
 
+  <properties>
+    <munit.version>2.3.14</munit.version>
+    <mule.maven.plugin.version>3.8.1</mule.maven.plugin.version>
+    <munit.extensions.maven.plugin.version>1.1.3</munit.extensions.maven.plugin.version>
+    <jacoco.maven.plugin.version>0.8.2</jacoco.maven.plugin.version>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.mule.tools.maven</groupId>
+        <artifactId>mule-maven-plugin</artifactId>
+        <version>${mule.maven.plugin.version}</version>
+        <extensions>true</extensions>
+      </plugin>
+
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>3.1.0</version>
+        <executions>
+          <execution>
+            <id>copy-munit-resources</id>
+            <phase>process-test-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/test-mule/munit</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src/test/munit</directory>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>com.mulesoft.munit.tools</groupId>
+        <artifactId>munit-maven-plugin</artifactId>
+        <version>${munit.version}</version>
+        <executions>
+          <execution>
+            <id>test</id>
+            <phase>test</phase>
+            <goals>
+              <goal>coverage-report</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <runtimeProduct>MULE_EE</runtimeProduct>
+          <runtimeVersion>${app.runtime}</runtimeVersion>
+          <redirectTestOutputToFile>true</redirectTestOutputToFile>
+          <coverage>
```

Comment:
Can this coverage stuff be removed and the test will still work? We don't care about coverage data yet. More generally - how much can this all be slimmed down to only be the bare minimum?

Reply:
This section is necessary. I don't know why the goal is "coverage-report".